dsl/src/commonMain/kotlin/it/unibo/collektive/field/operations/BooleanField.kt

```diff
@@ -0,0 +1,28 @@
+package it.unibo.collektive.field.operations
+
+import it.unibo.collektive.field.Field
+import it.unibo.collektive.field.Field.Companion.fold
+
+/**
+ * Check if all the [Boolean] elements in the field are true by default.
+ * If [predicate] is defined, it is applied to the elements.
+ * The local value is not considered, unless explicitly passed as [default].
```

Comment:
```suggestion
 * The local value is not considered, unless explicitly passed as [default].
```

If it is a fold, this not the expected behaviour

Reply:
I've changed the doc. Please, check if now is appropriate